tools/type_whisperer/BUILD

```diff
@@ -35,7 +35,38 @@ py_binary(
     ],
 )
 
+# Pack API type database text file into a char* string that can be referenced
+# at the C++ level.
+genrule(
```

Comment:
should we generate this cc in type_database rule in https://github.com/envoyproxy/envoy/pull/9276?

Reply:
Is there a big win from this? Since this `genrule` just gets invoked on the artifact from `type_database()` and is only ever consumed in one place (`api_type_db_lib`), it's easier to read/maintain if this is outside of a Bazel rule and we just use stock `genrule`.